metron-platform/metron-parsing/metron-parsing-storm/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -77,17 +85,6 @@ public ParserBolt( String zookeeperUrl
     super(zookeeperUrl);
     this.parserRunner = parserRunner;
     this.sensorToWriterMap = sensorToWriterMap;
```

Comment:
This was removed because the `WriterHandler.handleAck` method no longer exists.  Acking tuples is not the responsibility of the `StormBulkWriterResponseHandler` class.

Reply:
With HBaseWriter and PcapWriter now gone (they were legacy leftovers, I believe), is there a reason to keep MessageWriter around? If there are other side effects, e.g. Flux files, Ambari configs, docs, etc. I'm happy to have it left for now and make that a follow-on Jira, but figured it's worth mentioning.

Reply:
I experimented with this and the changes were just significant enough to make me want to push it off to a follow on.  If you can handle reviewing it here I'm happy to make the changes :)

Reply:
@merrimanr Sounds good to me - Would you mind creating a Jira for that and linking it to our comments here for reference and background?

Reply:
@merrimanr I may have missed the Jira/comment on it - did this ticket for the MessageWriter cleanup get created?

Reply:
https://issues.apache.org/jira/browse/METRON-2015